src/components/TodoList/TodoList.tsx

```diff
@@ -1 +1,17 @@
-export const TodoList = () => {};
+import { Todo } from '../../types/Todo';
+import { User } from '../../types/User';
+import { TodoInfo } from '../TodoInfo';
+
+interface Props {
+  todos: Todo[];
+  users: User[];
```

Comment:
The `users` prop is defined in the `Props` interface but is not used in the `TodoList` component. Consider removing it to simplify the code.